cluster/node.go

```diff
@@ -374,6 +378,12 @@ func (n *Node) Containers() []*Container {
 	return containers
 }
 
+func (n *Node) Images() []*dockerclient.Image {
+	n.RLock()
```

Comment:
The lock is useless since we are returning a reference to the list.

How about copying the values into a new list (as we do for `Containers()`)?